web/src/copy_and_paste.ts

```diff
@@ -402,6 +402,33 @@ export function paste_handler_converter(paste_html: string): string {
         .parseFromString(paste_html, "text/html")
         .querySelector("body");
     assert(copied_html_fragment !== null);
+
+    const para_elements = copied_html_fragment.querySelectorAll("p");
+
+    for (const element of para_elements) {
+        const children = [...element.childNodes];
+
+        for (let i = 0; i < children.length - 1; i += 1) {
+            const next_node = children[i + 1];
```

Comment:
But why are we still doing everything with this funny off-by-1 indexing and skipping the first child, instead of a simple `for…of` loop?